.github/workflows/conformance-aks-v1.10.yaml

```diff
@@ -53,10 +53,8 @@ concurrency:
     ${{
       (github.event_name == 'schedule' && github.sha) ||
       (github.event_name == 'issue_comment' && (
-        startsWith(github.event.comment.body, 'ci-aks-1.10') ||
-        startsWith(github.event.comment.body, '/ci-aks-1.10') ||
```

Comment:
Not very familiar with the details here, but is there code for handling of the followup text that we would need to remove as well since now there will be no trigger phrases for it?

Reply:
I don't think so. The followup text was never handled anywhere in the CI code as far as I know. It was free-form text, just to allow folks to document stuff.